### DISCUSSION TITLE
Integrating  changelog into the feed sidebar is amazing but needs a minor improvement

### DISCUSSION THREAD

**Discussion by thispsj** [community/community#17221] [Other Feature Feedback, Questions, & Ideas]
![Screenshot](https://user-images.githubusercontent.com/51746608/170051147-040adf4e-f245-466d-9b68-21b5ace44dcf.png)

Instead of View Changelog, could we have the possibility to load the rest of changelog into feed directly by pressing "More" button, which loads 5 entries at a time ?

**Comment by Stadeodo** [comment]
Sure! That will be great.

**Comment by Sechengine** [comment]
Alt ser if not else no merit please

**Comment by siminapasat** [comment]
Thank you for your feedback! It's a bit more complicated than a "minor improvement" , as the source of the data is completely different, but it's something we've been thinking about as well 👍
